composer.json

```diff
@@ -95,6 +95,9 @@
 		"typo3/cms-workspaces": "^9.5 || ^10.4 || ^11.5",
 		"typo3/coding-standards": "^0.5"
 	},
+	"suggest": {
+		"b13/container": "Just to be loaded after EXT:container"
```

Comment:
@kitzberger crazy stuff that the "suggest" influences the loading order of the extensions! I would have never expected it. And it really looks more like a hack.

I think that this should not be merged in this extension, but we should add a workaround (patch) in our project where we have both extensions active, for example by patching out the XCLASS of b13/containers.

Reply:
@baschny I have it already on the schedule/my list for v13.

Reply:
@sbuerk I just noted that b13/container checks for v12 or later and used new events instead of the hook. So this problem seems to happen only with TYPO3 v11, so... solved for the future, but not for our v11 project of course, but we find a workaround for that.